pipelines/install/metadata.obsah.yaml

```diff
@@ -0,0 +1,18 @@
+help: |
+  Run an install pipeline
+variables:
+  forklift_state:
+    parameter: --state
+    help: Forklift state to ensure
+    # TODO: up/rebuild/destroy
+  pipeline_os:
+    parameter: --os
+    help: Operating system to install, like centos7, debian10 or ubuntu1804
+  pipeline_type:
+    parameter: --type
+    help: Type of pipeline, like foreman, katello or luna
+    # TODO: choices: foreman/katello/luna
```

Comment:
Same here as above, but it does look like you want to eventually have something here for that?

Reply:
Currently we only have a set of variables for those 3. That means we can enforce just the 3 options.